controllers/bsl.go

```diff
@@ -252,6 +252,14 @@ func (r *DPAReconciler) updateBSLFromSpec(bsl *velerov1.BackupStorageLocation, d
 				bslSpec.Config["s3Url"] = s3Url
 			}
 		}
+
+		// Since the AWS SDK upgrade in velero-plugin-for-aws, data transfer to BSL bucket fails unless
+		// we specify the checksumAlgorithm. We will set it empty string if checksumAlgorithm is not specified by the user
+		// Setting it to an empty string will default to using the CRC32 algorithm for checksum calculation.
+		if _, exists := bslSpec.Config[checksumAlgorithm]; len(s3Url) > 0 && !exists {
+			bslSpec.Config["checksumAlgorithm"] = ""
```

Comment:
also use `checksumAlgorithm` var in here?

Reply:
Oversight, thank, fixed it!